### PR TITLE
zh: Update configure-a-tidb-cluster.md

### PR DESCRIPTION
**Pull request by pepezzzz**
add tikv level 2 parameter

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)

add typical tikv lever 2 parameter

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: zh/backup-restore-overview.md [hunk 1]
```diff
@@ -120,6 +120,10 @@ TiDB Operator 1.1 及以上版本推荐使用基于 CustomResourceDefinition (CR
 * `.spec.br.sendCredToTikv`：BR 进程是否将自己的 AWS 权限 或者 GCP 权限传输给 TiKV 进程。默认为 `true`。
 * `.spec.br.options`：BR 工具支持的额外参数，需要以字符串数组的形式传入。自 v1.1.6 版本起支持该参数。可用于指定 `lastbackupts` 以进行增量备份。
 
+### toolImage 字段介绍
+
+* `.spec.toolImage`：代表 BR 工具使用的二进制镜像，默认使用 pingcap/br:${tikv_version}。
```
Comment (csuzhangxc) [inline zh/backup-restore-overview.md:125]: 前面 `通用字段介绍` 介绍里有了，这里还要加吗？

File: zh/backup-restore-overview.md [hunk 2]
```diff
@@ -120,6 +120,10 @@ TiDB Operator 1.1 及以上版本推荐使用基于 CustomResourceDefinition (CR
...
+
```
Comment (DanielZhangQD) [inline zh/backup-restore-overview.md:126]:
```suggestion
```

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Review by qiancai — APPROVED** [review]

**Pull request merged**